### ISSUE TITLE
[Error] Missing JSDoc

### ISSUE THREAD

**Issue by madcampos** [eslint/typescript-eslint-parser#116]
**What version of TypeScript are you using?**
2.0.9

**What version of `typescript-eslint-parser` are you using?**
1.0.0

**What code were you trying to parse?**

```javascript
/*eslint "require-jsdoc": "error"*/

/**
 * It returns 10
 */
function foo() {
    return 10;
}
```

**What did you expect to happen?**
Code pass validation.

**What happened?**
Error: Missing JSDoc

**Comment by JamesHenry** [comment]
Comments are not available as a configuration source yet, this is a known issue being tracked for this project here https://github.com/eslint/typescript-eslint-parser/issues/5.

The underlying solution is being worked on in ESLint core here https://github.com/eslint/eslint/issues/6724
